go.mod

```diff
@@ -5,11 +5,10 @@ go 1.13
 require (
 	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
-	github.com/container-storage-interface/spec v1.1.0
+	github.com/container-storage-interface/spec v1.2.0
```

Comment:
@kayrus  we also bump CSI driver version with CSI Version
https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/csi/cinder/driver.go#L36

Reply:
This is weird that it is not possible to reuse the version from the package. Updated.